services/web-app/package.json

```diff
@@ -56,6 +56,7 @@
     "prop-types": "^15.8.1",
     "query-string": "^7.1.1",
     "react": "^17.0.2",
+    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
This could use the same strategy as [this](https://github.com/carbon-design-system/carbon-platform/pull/638#discussion_r861983221) to avoid a lib dependency.